docs/2-recurring-billing-module/README.md

```diff
@@ -107,25 +130,71 @@ It will allow your module to automatically install PsAccount when not available,
 
 ```php
 class Rbm_example extends Module {
+    /**
+     * @var ServiceContainer
+     */
+    private $container;
+
+    public function __construct()
+    {
+        // ...
+
+        if ($this->container === null) {
+            $this->container = new \PrestaShop\ModuleLibServiceContainer\DependencyInjection\ServiceContainer(
+                $this->name,
+                $this->getLocalPath()
+            );
+        }
+    }
+
     // ...
     public function install()
     {
         // Load PS Account utility
-        return parent::install() && $this->getService('ps_accounts.installer')->install();
+        return parent::install() &&
+            $this->getService('ps_accounts.installer')->install();
     }
 
     // ...
 
+    /**
+     * Retrieve service
+     *
+     * @param string $serviceName
+     *
+     * @return mixed
+     */
     public function getService($serviceName)
     {
         return $this->container->getService($serviceName);
     }
-
 }
 ```
 
 > You should follow the documentation from [prestashop-accounts-installer](https://github.com/PrestaShopCorp/prestashop-accounts-installer) to properly install the PS Account utility.
 
+Like your module, you need to register PsAccount as Service
```

Comment:
j'enlèverai le "Like your module," et juste mettre "You need"

Reply:
Effectivement, c'est peut-être mieux 👍